File: rb/test/exists_test.rb
# Gitlab SDK exists test

require "minitest/autorun"
require_relative "../Gitlab_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = GitlabSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Gitlab SDK exists test

require_once __DIR__ . '/../gitlab_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = GitlabSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
